The library does not support Storybook 7

**Describe the bug**
I am using Storybook 7 in my project, and when I try to install the library I receive an error (Could not resolve dependency:
npm ERR! peer storybook@">=6.4.19 <7" from storybook-xstate-addon@2.3.7) 

**To Reproduce**
* Storybook Version: 7.0.18
* This Plugin Version: 2.3.7
* Include a [Replay.io recording](https://www.replay.io/) with comments to demonstrate the bug. If this is not possible please provide a reproduction of the issue in a repository.

**Expected behavior**
The library should be installed

**Additional context**
Add any other context about the problem here.
![image](https://github.com/SimeonC/storybook-xstate-addon/assets/84324716/8e092197-cc01-45e1-b495-fa8250129cf7)


Obs.: On the same day I had a problem with a project but on another, I don't. for booths, I used the storybook 7.0.18.

project 1 - (error) - packege.json:

```
{
  "private": true,
  "scripts": {
    "storybook": "cross-env ENV=local storybook dev -p 6006",
    "build-storybook": "storybook build"
  },
  "devDependencies": {
    "@babel/preset-env": "^7.21.5",
    "@babel/preset-react": "^7.18.6",
    "@playwright/test": "^1.33.0",
    "@storybook/addon-a11y": "^7.0.18",
    "@storybook/addon-essentials": "^7.0.18",
    "@storybook/addon-interactions": "^7.0.18",
    "@storybook/addon-links": "^7.0.18",
    "@storybook/blocks": "^7.0.18",
    "@storybook/react": "^7.0.18",
    "@storybook/react-webpack5": "^7.0.18",
    "@storybook/testing-library": "^0.1.0",
    "@types/node": "^20.2.1",
    "autoprefixer": "^10.4.14",
    "cross-env": "^7.0.3",
    "css-loader": "^6.7.4",
    "mini-css-extract-plugin": "^2.7.6",
    "postcss": "^8.4.23",
    "postcss-loader": "^7.3.0",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "storybook": "^7.0.18",
    "tailwindcss": "^3.3.2"
  },
  "name": "test-error",
  "packageManager": "npm@9.5.1",
  "workspaces": [
    "../packages/*"
  ],
  "dependencies": {
    "@tailwindcss/forms": "^0.5.3",
    "webpack-cli": "^4.10.0",
    "webpack-dev-server": "^4.15.0"
  }
}
```

project 2 - (not error) - package.json:

```
{
  "name": "react_xstate",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject",
    "storybook": "storybook dev -p 6006",
    "build-storybook": "storybook build"
  },
  "dependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-scripts": "5.0.1",
    "typescript": "^4.9.5",
    "xstate": "^4.37.2"
  },
  "devDependencies": {
    "@storybook/addon-essentials": "^7.0.18",
    "@storybook/addon-interactions": "^7.0.18",
    "@storybook/addon-links": "^7.0.18",
    "@storybook/blocks": "^7.0.18",
    "@storybook/preset-create-react-app": "^7.0.18",
    "@storybook/react": "^7.0.18",
    "@storybook/react-webpack5": "^7.0.18",
    "@storybook/testing-library": "^0.0.14-next.2",
    "@testing-library/jest-dom": "^5.16.5",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "@types/jest": "^27.5.2",
    "@types/node": "^16.18.34",
    "@types/react": "^18.2.7",
    "@types/react-dom": "^18.2.4",
    "@xstate/inspect": "^0.7.1",
    "@xstate/react": "^3.2.2",
    "babel-plugin-named-exports-order": "^0.0.2",
    "eslint-plugin-storybook": "^0.6.12",
    "prop-types": "^15.8.1",
    "storybook": "^7.0.18",
    "storybook-xstate-addon": "^2.3.7",
    "webpack": "^5.85.0"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest",
      "plugin:storybook/recommended"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}

```

project 2 - success print
![image](https://github.com/SimeonC/storybook-xstate-addon/assets/84324716/8c34121b-8afb-404c-9e9f-f98ca9412825)

--- a/package.json
+++ b/package.json
@@ -68,16 +68,16 @@
     "xstate": "4.32.1"
   },
   "peerDependencies": {
-    "@storybook/addons": ">=6.4.19 <7",
-    "@storybook/api": ">=6.4.19 <7",
-    "@storybook/components": ">=6.4.19 <7",
-    "@storybook/core-events": ">=6.4.19 <7",
-    "@storybook/theming": ">=6.4.19 <7",
-    "@xstate/inspect": "^0.6 || ^0.7",
+    "@storybook/addons": ">=6.4.19 <8",
+    "@storybook/api": ">=6.4.19 <8",
+    "@storybook/components": ">=6.4.19 <8",
+    "@storybook/core-events": ">=6.4.19 <8",
+    "@storybook/theming": ">=6.4.19 <8",
+    "@xstate/inspect": ">=0.6 <0.9",
     "@xstate/react": ">=2 <4",
     "react": ">=16.8.0 <19",
     "react-dom": ">=16.8.0 <19",
-    "storybook": ">=6.4.19 <7",
+    "storybook": ">=6.4.19 <8",
     "xstate": "^4.18.0"
   },
   "publishConfig": {

